### PR TITLE
[DPCP-8]: Google OAuth

### PR DESCRIPTION
**Pull request by angeloreale**
PLEASE REVIEW YOUR OWN PR BEFORE OPENING/UN-DRAFTING IT

# ar(feat) DPCP-8: Google OAuth

This PR adds Google OAuth to Nyx+Hypnos.

## Have you?

- [x] Reviewed your own PR?
- [ ] Tested your own feature/fix meets Acceptance Criteria?
- [x] Made sure it passes all checks? (lint, build, integrations, scripts, etc)?
- [ ] Added screenshots? (if relevant)
- [ ] Added unit tests? (if necessary)
- [ ] Added documentation? (if necessary)

If so, THANKS! You can pop a beer/soda.

Please open your PR and ping your colleagues to review it on #web-devs channel.

Also, make sure to remind them on a daily basis during stand-up.

### CODE REVIEW

**Comment by angeloreale** [comment]
Need deployment to test via Nyx.

**Comment by angeloreale** [comment]
This worked via Preview link on Nyx after deploying Hypnos. It is erroring in prod. Not willing to debug now.

**Comment by angeloreale** [comment]
tl;dr this exact config/code works for all providers on another project. Won't go into a pointless rabbit hole.
